src/containers/Pager.js

```diff
@@ -19,16 +20,24 @@ const mapDispatchToProps = dispatch => bindActionCreators({
   refreshData: actions.refreshData
 }, dispatch)
 
-const handlers = {
+const changePageHandlers = {
   changePage: props => event => {
     const nextPage = event.target.getAttribute('data-page')
     props.refreshData(props.name, {...props.filter, page: nextPage})
   }
 }
 
+const mapNumbersPagesHandlers = {
+  mapNumberedPagerButtons: props => (page, index) =>
+    <PagerButton loading={props.loading} key={index} page={page} currentPage={props.filter.page} changePage={props.changePage}>
+      {page + 1}
+    </PagerButton>
```

Comment:
I don't think we should include any UI logic in the containers.  I can think of two possible solutions:
1. create a PageButtons component and container that could more naturally handle this
2. transform this into a `getButtons` handler that would return a data object per button with which the component could map over